src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,13 @@
-export const Tabs = () => {};
+export const Tabs = ({
+  tab: { id = 'tab-1', title, content }, setValue, value,
+}) => (
+  <li data-cy="Tab" className={value === id ? 'is-active' : ''}>
+    <a
+      href={`#${id}`}
+      data-cy="TabLink"
+      onClick={() => setValue(id)}
```

Comment:
The onClick function should prevent the default action to avoid unnecessary page refresh when the anchor is clicked. Consider adding event.preventDefault() to the onClick function.